README.md

```diff
@@ -89,6 +89,21 @@ else:
     paginated = False
 ```
 
+#### Retries and Backoff
+
+By default the client uses [this](https://learn.microsoft.com/en-us/python/api/azure-core/azure.core.pipeline.policies.retrypolicy?view=azure-python)
```

Comment:
Since we use a somewhat well known retry policy given our usage of Microsoft's client generator, I think it's worth mentioning it explicitly for the folks who might be aware of it. Just an idea though – I don't feel too strongly about this, so if you disagree, you don't have to accept the suggestion.
```suggestion
By default the client uses the same retry policy as the [Azure SDK for Python](https://learn.microsoft.com/en-us/python/api/azure-core/azure.core.pipeline.policies.retrypolicy?view=azure-python).
```

Reply:
I think that's a good suggestion, will commit change- thanks!